packages/shared/babel-ast-utils/src/index.js

```diff
@@ -36,6 +36,8 @@ export async function parse({
         strictMode: false,
         sourceType: 'module',
         plugins: ['exportDefaultFrom', 'exportNamespaceFrom', 'dynamicImport'],
+        // $FlowFixMe
+        startLine: asset.meta.loc?.start.line,
```

Comment:
This offsets location information for both error messages and source maps of inline JS assets inside HTML so it starts at the correct line number of the original file